qt/python/mantidqt/project/projectloader.py

```diff
@@ -0,0 +1,52 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2017 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantidqt package
+#
+import json
+
+import workspaceloader
+from mantid import AnalysisDataService as ADS
+from mantid import logger
+
+
+ENCODED_FILE_NAME = "mantidsave.project"
```

Comment:
As this stuff is in the `mantidqt` package it should probably not mandate the filename used. I think we should define this in the workbench, maybe in the `config` package? We can then pass this in to the new `Project` class constructor and subsequently to the other classes down here. It feels like the type of thing that should be parameterized.

Reply:
Done, but currently it is just defined as a string in workbench.py. Instead of being defined in the config package, as I was unsure of implementation of configservice/equivalent in workbench.